specs/electra/beacon-chain.md

```diff
@@ -1022,9 +1022,8 @@ def verify_and_notify_new_payload(self: ExecutionEngine,
 
     # [Modified in Electra]
     if not self.notify_new_payload(
-            execution_payload, 
-            execution_requests, 
-            parent_beacon_block_root):
+            parent_beacon_block_root,
+            execution_requests):
```

Comment:
Just realized this mistake. I accidentally deleted `execution_payload` here 🤦

Reply:
whoops

Reply:
For reference, it was fixed in here:

* #3971

Reply:
> Just realized this mistake. I accidentally deleted `execution_payload` here 🤦

devops199?